holidays/calendars/hindu.py

```
@@ -1207,6 +1461,45 @@ class _HinduLunisolar:
+        2025: (JUN, 7),
+        2026: (JUN, 25),
+        2027: (JUN, 15),
+        2028: (JUN, 3),
+        2029: (JUN, 22),
+        2030: (JUN, 12),
+        2031: (JUN, 2),
+        2032: (JUN, 19),
+        2033: (JUN, 9),
+        2034: (MAY, 28),
+        2035: (JUN, 16),
+    }
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)